README.md

```diff
@@ -151,10 +151,12 @@ const [err, user] = await fastify.to(
 ```
 
 #### Custom error handler
-This plugins also adds a custom error handler which hides the error message in case of `500` errors, instead it returns `Something went wrong`.<br>
+This plugins also adds a custom error handler which hides the error message in case of `500` errors that are being generated by throwd errors, instead it returns `Something went wrong`.<br>
```

Comment:
```suggestion
This plugins also adds a custom error handler which hides the error message in case of `500` errors that are being generated by throwed errors, instead it returns `Something went wrong`.<br>
```